client/internal/clientcommon.go

```diff
@@ -369,3 +375,47 @@ func (c *ClientCommon) SetPackageStatuses(statuses *protobufs.PackageStatuses) e
 
 	return nil
 }
+
+// SetCustomCapabilities sends a message to the Server with the new custom capabilities.
+func (c *ClientCommon) SetCustomCapabilities(customCapabilities *protobufs.CustomCapabilities) error {
```

Comment:
Is this func used anywhere? I can't find any usage.

Reply:
I'm not sure how I missed this. Added to the Client interface, implemented and tested.